src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const styleStringToArr = sourceString.trim().split(';');
+
+  for (const key of styleStringToArr) {
+    const styleToArr = key.trim().split(':');
+
+    if (styleToArr[1] && styleToArr[0] !== undefined) {
+      result[styleToArr[0].trim()] = styleToArr[1].trim();
+    }
```

Comment:
Avoid using loops in this task. Instead, use array iteration methods like `map` and `filter` to transform the array of style declarations into an object.